0000-01-02-darkfax.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
created a branch called my_slde, created a file within it and made a commit, and opened a pull request